onnxruntime/core/framework/session_state.cc

```diff
@@ -1005,9 +1011,9 @@ Status SessionState::FinalizeSessionStateImpl(const std::basic_string<PATH_CHAR_
 #endif
 
   // Memory pattern tracer allocates all initializers on a single continous
-  // buffer. This has the effect of reducing memory fragementation. 
+  // buffer. This has the effect of reducing memory fragementation.
   // Further more, NCCL kernels require initializers to be allocated
-  // continously. 
+  // continously.
```

Comment:
Typo: s/continously/continuously/ (but it should probably be "contiguously").

Reply:
It is contiguously. Thanks!

---
In reply to: [600934394](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r600934394) [](ancestors = 600934394)